$OpenBSD$

# Remove -Wredundant-decls which is rather noisy with OpenBSD system headers.

--- cmake/GammuTuneCompiler.cmake.orig	Tue Oct 13 13:44:09 2015
+++ cmake/GammuTuneCompiler.cmake	Tue Oct 13 13:44:24 2015
@@ -2,7 +2,6 @@
 # used for whole tree (eg. Python module)
 
 MACRO_TUNE_COMPILER("-Wwrite-strings")
-MACRO_TUNE_COMPILER("-Wredundant-decls")
 if (DEBUG)
     MACRO_TUNE_COMPILER("-fstrict-aliasing")
 endif (DEBUG)
